.claude/commands/local/audit-skill.md

```diff
@@ -0,0 +1,182 @@
+---
+description: Audit a skill against Anthropic skill-creator standards
+argument-hint: <skill-name-or-path>
+---
+
+# /local:audit-skill
+
+Audit a skill against the Anthropic skill-creator standards and provide structured recommendations for improvement.
+
+## Usage
+
+```bash
+# By skill name (looks in .claude/skills/)
+/local:audit-skill dignified-python
+
+# By file path
+/local:audit-skill /path/to/skill/SKILL.md
+```
+
+---
+
+## Agent Instructions
+
+Execute the skill audit workflow following these phases carefully.
+
+### Phase 1: Resolve Target Skill
+
+Determine the skill to audit based on the argument provided: `$ARGUMENTS`
+
+**Resolution logic:**
+
+1. If the argument contains `/`, treat it as a file path
+   - If it ends with `SKILL.md`, use the parent directory
+   - Otherwise, assume it's the skill directory and look for `SKILL.md` inside
+2. If the argument does not contain `/`, treat it as a skill name
+   - Look for `.claude/skills/<skill-name>/SKILL.md`
+
+**Validation:**
+
+- Confirm the resolved path exists
+- If not found, display an error message and stop:
+  ```
+  Error: Skill not found at <resolved-path>
+  ```
+
+### Phase 2: Fetch Standards from GitHub
+
+Fetch the authoritative skill-creator standards from Anthropic's GitHub repository.
+
+**Use WebFetch to retrieve these files:**
+
+1. `https://raw.githubusercontent.com/anthropics/skills/main/skills/skill-creator/SKILL.md`
+   - Prompt: "Extract all the skill creation guidelines, requirements, and best practices"
+
+2. `https://raw.githubusercontent.com/anthropics/skills/main/skills/skill-creator/references/output-patterns.md`
+   - Prompt: "Extract output pattern guidelines and examples"
+
+3. `https://raw.githubusercontent.com/anthropics/skills/main/skills/skill-creator/references/workflows.md`
+   - Prompt: "Extract workflow pattern guidelines and examples"
+
+**Error handling:**
```

Comment:
make this more future-proff to skill restructures.   
  
https://raw.githubusercontent.com/anthropics/skills/main/skills/skill-creator  
  
look at this at list all files, or look at the web ui

Reply:
Updated Phase 2 to dynamically discover files from the GitHub repository structure instead of hardcoding URLs. Added Step 2.1 (browse directory to discover files) and Step 2.2 (fetch discovered .md files). Also added fallback to known files if directory listing fails. Resolved via /erk:pr-address at 2025-12-30 09:20 PM PST

_Addressed via `/erk:pr-address` at 2025-12-30 21:20 PST_